tests/lightning_tensor/test_gates_and_expval.py

```diff
@@ -175,12 +176,14 @@ def circuit_ansatz(params, wires):
         (qml.ops.prod(qml.X(0), qml.Y(1))),
     ],
 )
-def test_integration_for_all_supported_gates(returns):
+def test_integration_for_all_supported_gates(returns, tn_backend):
```

Comment:
what about using `method` to make it aligned with the UI?

Reply:
Done!